bower.json

```diff
@@ -1,17 +1,18 @@
 {
-  "name": "preprint-service",
-  "dependencies": {
-    "ember": "~2.7.0",
-    "ember-cli-shims": "0.1.1",
-    "ember-qunit-notifications": "0.1.0",
-    "osf-style": "https://github.com/CenterForOpenScience/osf-style.git#5dcf5997ad2808ffc7c8564f94cde102bf7a4a96",
-    "font-awesome": "~4.6.3",
-    "jquery": "^2.2.4",
-    "pretender": "~1.1.0",
-    "Faker": "~3.1.0",
-    "bootstrap-treeview": "^1.2.0",
-    "dropzone": "^4.3.0",
-    "bootstrap-sass": "^3.3.6",
-    "toastr": "^2.1.2"
-  }
+    "name": "preprint-service",
```

Comment:
Most JS tools indent things by 2 characters by default, but our style guide calls for 4.

Usually, we leave bower and package.json alone, because the tools that auto-regenerate those files do unpredictable things with indentation and don't always obey our rules. In general we just try to match the indentation that was already there to avoid noisy PRs.